ArduPlane/system.cpp

```diff
@@ -225,6 +225,40 @@ static bool mode_reason_is_landing_sequence(const ModeReason reason)
 }
 #endif // AP_FENCE_ENABLED
 
+// Check if this mode can be entered from the GCS
```

Comment:
```suggestion
#if GCS_MODE_ENTRY_FILTERING_ENABLED
// Check if this mode can be entered from the GCS
```

etc

this should also be in `GCS_Plane.cpp`

Reply:
We decided on the call that it was OK where it was. Do you still want a define? I guess it should live in AP_Vehicle?